go.mod

```diff
@@ -4,17 +4,19 @@ go 1.15
 
 replace github.com/btcsuite/btcd => github.com/lbryio/lbrycrd.go v0.0.0-20200203050410-e1076f12bf19
 
+//replace github.com/lbryio/reflector.go => /home/niko/go/src/github.com/lbryio/reflector.go
```

Comment:
this is for when I work on both the player and reflector.go at the same time locally so I don't have to keep pushing and updating go.mod

changes are immediate.

I should check if goland supports this in a way so that I don't have to put it in here. it's a very handy feature